gms/versioned_value.hh

```diff
@@ -81,6 +81,7 @@ public:
     static constexpr const char* REMOVING_TOKEN = "removing";
     static constexpr const char* REMOVED_TOKEN = "removed";
 
+    static constexpr const char* STATUS_PREPARE_REPLACE = "PREPARE_REPLACE";
```

Comment:
Do we need to qualify it with a cluster feature? So old nodes don't see the new value?

Replace and similar aren't supported during upgrade, but rewarding someone who breaks the rules (perhaps accidentally) with a bigger problem isn't a good idea.

Reply:
We do not need a gossip feature because old node will just ignore the new value.